client/components/TestRenderer/AssertionsFieldset/index.jsx

```diff
@@ -28,18 +29,30 @@ const AssertionsFieldset = ({ assertions, commandIndex, assertionsHeader }) => {
       </legend>
       {assertions.map((assertion, assertionIndex) => {
         const { description, passed, click } = assertion;
-
         return (
-          <Label key={`AssertionKey_${assertionIndex}`}>
-            <input
-              type="checkbox"
-              id={`pass-${commandIndex}-${assertionIndex}`}
-              name={`assertion-${commandIndex}-${assertionIndex}`}
-              defaultChecked={passed}
-              onClick={click}
-            />
-            {description[0]}
-          </Label>
+          <Fieldset key={`AssertionKey_${assertionIndex}`}>
```

Comment:
Looks good to me but could the radio buttons also be aligned horizontally? Like it is in https://github.com/w3c/aria-at/pull/1090

Reply:
Good call! Did this here: [6079e4c](https://github.com/w3c/aria-at-app/pull/1161/commits/6079e4ceccb8a0811ade14c5519461b50383277a)